lib/matplotlib/axes/_base.py

```diff
@@ -3426,12 +3431,13 @@ def set_xticklabels(self, labels, fontdict=None, minor=False, **kwargs):
                 'verticalalignment': 'baseline',
                 'horizontalalignment': loc}
 
-        minor : bool, optional
+        minor : bool, default: False
             Whether to set the minor ticklabels rather than the major ones.
 
         Returns
         -------
-        A list of `~.text.Text` instances.
+        labels
+            A list of `~.text.Text` instances.
```

Comment:
I would rather write this as
```
list of `~.text.Text`
	The labels.
```
because "labels" isn't an API name.

Take it or leave it...